print([[........................................................................................
....................................................................................................
.................................................::.................................................
............................................:^^~!!!!~^^:............................................
..........................................:~!!!~~~~~!!!!~:..........................................
........................................:!!!~~~~~~~~~~~~!!~:........................................
.......................................:77~~~~~~~~~~~~~~!!7!:.......................................
......................................:7!~!~~~~~~~~~~~~~~!~7!:......................................
......................................^?~!!~~~~~~~!~~~~~~!!~J^......................................
...........................:^~~!!!!~~~!?~!!!~~~~~!!~~~~~!!!~J~~~~!!!!~~^:...........................
.........................:^~!!~~~~~~!!!J!!!!!~~~~!!~~~~!!!7!!!!!~~~~~~~~~^:.........................
.......................:~!~~~~~~~~~~!!7??!!!!!~~~!!~~~!!7!!~~!!!~~~~~~~~~~!~:.......................
......................:7!~~~~~~~~~~~~~!7??!~!!!!!!!!!!!!!~!!!~~~~~~~~~~~~~~!7:......................
......................7!~~~~~~~~~~~~~~~!!??7!!7!!~!!77!!!!!~~~~~~~~~~~~~~~~~7!......................
......................!!~~~~~~~~~~~~~~~~~!7??7!7^^~~7!77!!!~~~~~~!!~~~~~~~~!J!......................
......................:?!~~~~~~~~~~~~!!!!!77~!!7!!7~!!!^!7!!!!!~~~~~~~~~~~!7?:......................
.......................^?!!~~~~~~~~~~~~~~!!7!!!!?YJ?~!7!??7!!!!~~~~~~~~!!!7J^.......................
........................^7?7!!!~~~~~~~!!!!!7??7~!?7!~!7777777777!!!!!!777?7^........................
..........................^7??77!!!!!!!!777??7^~!7~!!^!?7!!!!!~~77?????7!^..........................
............................::!????????????7??77??~?7!!?!!!!!!!!~~7?7~::............................
.............................:!7!77777!!!!!!777!?7~77777!!~~~!!!!!~~!7..............................
.............................~?~!!!~~~~~~!!!!~~!?7~!!!!!!!!~~~~~~~~!~?!.............................
............................:?~!!~~~~~~~!!!~~~~!??~!!!~~~!!!~~~~~~~~~!J^............................
............................!7~!~~~~~~~!~~~~~~!!?J!~!!!~~~~!!~~~~~~~!!J7............................
............................~7~!~~~~~~~~~~~~~~!77J?~!!!~~~~~~~~~~~~~!!J~............................
.............................7!~~~~~~~~~~~~~!!!77JJ?!!!!~~~~~~~~~~~~!??.............................
...........................::~77~~~~~~~~~~!!!77??77?J7!!!!!~~~~~~~!7?7:.............................
.......................::.^~!!!???7!!!~~!7??JJJ7!!!!7?JJ??7!!!!!7??J7~^^::.:........................
........................::^~!~!!!77??JJJJ??77!!!!!!!!!!77??JJJJ??77!!!~^:::.:.......................
............................::::^~~~~!!!!~~~~^^::::::^^~~~~!!!~~~~^^::::.:..........................
....................................::...:::............:...........................................
....................................................................................................
]])

if not game:IsLoaded() then
    game.Loaded:Wait()
end

repeat
    task.wait()
until game.Players.LocalPlayer.Character and game.Players.LocalPlayer.Character:FindFirstChild("Humanoid")

local Game = {
    [96645548064314] = "https://auth.syscure.vip/obf/ba942cb1b4f3b9a9449b4796a71a85e1.lua"
}

local Modules = {
    Executor = ((identifyexecutor and identifyexecutor()) or (getexecutorname and getexecutorname()) or "Unknown"),
    Load = function()
        local src = Game[game.PlaceId]
        if not src then
            return warn("❌ Unsupported game ->", game.PlaceId)
        end;
        loadstring(game:HttpGet(src))()
    end;
}

local NotSupport = function()
    local exec = string.lower(Modules.Executor)
    for _, v in ipairs({
        "xeno",
        "solara",
        "jjsploit"
    }) do
        if exec == v or string.find(exec, v) then
            return true
        end;
    end;
    return false
end;

print("Execute On ->", Modules.Executor)

if NotSupport() then
    game.Players.LocalPlayer:Kick( "⚠️ Executor " .. Modules.Executor .. " not supported ⚠️" )
    return
end;

Modules.Load()
